src/Libraries/Module/Templates/Demo/Web/Controllers/AccountController.tpl

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Quantum PHP Framework
+ *
+ * An open source software development framework for PHP
+ *
+ * @package Quantum
+ * @author Arman Ag. <arman.ag@softberg.org>
+ * @copyright Copyright (c) 2018 Softberg LLC (https://softberg.org)
+ * @link http://quantum.softberg.org/
+ * @since 2.9.0
+ */
+
+namespace {{MODULE_NAMESPACE}}\Controllers;
+
+use Shared\Transformers\PostTransformer;
+use Quantum\Factory\ServiceFactory;
+use Quantum\Factory\ViewFactory;
+use Shared\Services\AccountService;
+use Shared\Services\PostService;
+use Quantum\Http\Response;
+use Quantum\Http\Request;
+use Quantum\Factory\ModelFactory;
+use Shared\Models\User;
+use Quantum\Libraries\Hasher\Hasher;
+use Quantum\Libraries\Auth\Contracts\AuthenticatableInterface;
+
+/**
+ * Class AccountController
+ * @package Modules\Web\Controllers
+ */
+class AccountController extends BaseController
+{
+    /**
+     * Main layout
+     */
+    const LAYOUT = 'layouts/main';
+
+    /**
+     * Account profile
+     */
+    const ACCOUNT_PROFILE = '#account_profile';
```

Comment:
Those are not that important to mentioned in here, can be just appended to the redirect URL